core/server/api/v2/utils/validators/input/schemas/posts-add.json

```diff
@@ -0,0 +1,20 @@
+
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "posts.add",
+    "title": "posts.add",
+    "description": "Schema for posts.add",
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+        "posts": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "allOf": [{"$ref": "posts#/definitions/post"}],
```

Comment:
This is the syntax that is available to have both original schemas and extended properties ("required" from below). In case there is a need for deeper base schema definition modifications [JSON Merge Patch](https://tools.ietf.org/html/rfc7396) could be applied (stayed away from overcomplicating things for now)